sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeTableCommand.scala

```diff
@@ -108,4 +126,8 @@ case class AnalyzeTableCommand(tableName: String) extends RunnableCommand {
 
 object AnalyzeTableCommand {
   val TOTAL_SIZE_FIELD = "totalSize"
+  // same as org.apache.hadoop.hive.common.StatsSetupConst
+  val ROW_COUNT = "numRows"
+  val STATS_GENERATED_VIA_STATS_TASK = "STATS_GENERATED_VIA_STATS_TASK"
+  val TRUE = "true"
```

Comment:
Agree. It is not case sensitive. I tried it using the Hive command line. It works well even if it is `TrUe`